components/mpas-ocean/src/mode_init/mpas_ocn_init_global_ocean.F

```diff
@@ -893,7 +893,8 @@ subroutine ocn_init_setup_global_ocean_create_model_topo(domain, iErr)!{{{
              end if
              if (isOcean) then
                 ! Enforce minimum depth
-                bottomDepth(iCell) = max(bottomDepthObserved(iCell), refBottomDepth(minimum_levels))
+                bottomDepth(iCell) = max(bottomDepthObserved(iCell), config_global_ocean_minimum_depth)
+                bottomDepth(iCell) = min(bottomDepth(iCell), refBottomDepth(minimum_levels))
```

Comment:
This should have been a `max`, not a `min`.

Reply:
Thanks for fixing this @xylar! Sorry I overlooked this change and didn't catch it earlier.